### PR TITLE
Feature/update background color of the top area of the timetable screen

### LINKED ISSUE #560: Update background color of the top area of the timetable screen.
**Idea Description**
Please update background color of timetable screen so that we can align we figma.

**Reference images and links**
スクリーンショット 2023-08-16 21 12 19

### PR DESCRIPTION
**Pull request by Masaki-U**
## Issue
- close #560

## Overview (Required)
- update background color of the top area of the timetable screen
- TopAppBar title icon now change color in light and dark mode respectively.

## Link
- [Light](https://www.figma.com/file/MbElhCEnjqnuodmvwabh9K/DroidKaigi-2023-App-UI?type=design&node-id=56044-41124&mode=dev)
- [Dark](https://www.figma.com/file/MbElhCEnjqnuodmvwabh9K/DroidKaigi-2023-App-UI?type=design&node-id=56145-55124&mode=dev)

## Screenshot
Before | After
:--: | :--:
 | 
 |

### CODE REVIEW

**Comment by momomomo111** [comment]
@Masaki-U  Thank you for your contribution!

Could you please change the title of the Pull Request to make it more understandable?

File: feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/component/TimetableTopArea.kt [hunk 1]
```diff
@@ -34,7 +33,7 @@ fun TimetableTopArea(
 ) {
     TopAppBar(
         title = {
-            Image(
+            Icon(
```
Comment (takahirom) [inline feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/component/TimetableTopArea.kt:36]: 👍

File: feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/TimetableScreen.kt [hunk 2]
```diff
@@ -85,6 +100,28 @@ data class TimetableScreenUiState(
     val contentUiState: TimetableSheetUiState,
 )
 
+private val timetableTopBackgroundLight = Color(0xFFF6FFD3)
+private val timetableTopBackgroundDark = Color(0xFF2D4625)
+
+@Composable
+@ReadOnlyComposable
```
Comment (takahirom) [inline feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/TimetableScreen.kt:107]: 🆒

**Review by takahirom — APPROVED** [review]
Thanks for your cool pull request!

**Pull request merged**